zstd.go

```diff
@@ -161,3 +161,18 @@ func Decompress(dst, src []byte) ([]byte, error) {
 	defer r.Close()
 	return ioutil.ReadAll(r)
 }
+
+// DecompressInto decompresses src into dst. Unlike Decompress, DecompressInto
+// requires that dst be sufficiently large to hold the decompressed payload.
+// DecompressInto may be used when the caller knows the size of the decompressed
+// payload before attempting decompression.
+//
+// If dst is too small, DecompressInto returns an error.
+func DecompressInto(dst, src []byte) error {
```

Comment:
to make it a bit closer to golang stdlib, I'd probably have the method signature also returns the nb of bytes written (`DecompressInto(dst, src []byte) (int, error)` like https://pkg.go.dev/io#Copy). You can always ignore it for your use-case but I think it will usage a bit more wide at no extra cost

Reply:
Makes sense, done.